joomla/joomla-cms
# Make menu item alias type generate a correct alias from title

Pull Request for New issue.
#### Summary of Changes

The goal of this PR is to solve a long standing bug of the Menu item alias.
When you create a menu item of type alias, the alias field is auto-generated with the system date and not with the menu title.

This is very annoying since we have to manually edit it after or we get the menu item children URI with that date.

![image](https://cloud.githubusercontent.com/assets/9630530/14615177/58881eb8-059c-11e6-86be-cca5de9468ae.png)
#### Testing Instructions
1. Use latest staging
2. Create a menu item of type "Menu Item Alias" and let it generate the alias field.
   You'll notice it generate a alias with the current date, instead of an alias from the title (as the other menu items)
3. Apply patch
4. Repeat step "2."
   You'll notice it will generate a alias from the title.
#### Observations

I'm not yet very comfortable with the JTable, so if a maintainer can review this would be appreciated (is just one line of code that changes).
